core/collector.class.inc.php

```diff
@@ -265,16 +265,17 @@ protected function GetCharset()
 	protected function GetVersionFromModuleFile()
 	{
 		$aFiles = glob(APPROOT.'collectors/module.*.php');
-		$sModuleFile = null;
-		$sModuleFile = reset($aFiles);
-		if ($sModuleFile == null)
+		if (!$aFiles)
```

Comment:
Should we write this if ($aFiles === false) ?

Reply:
I think it can also be an empty array in some cases..

Reply:
You agree?

Reply:
@dflaven?

Reply:
yes you are right regarding glob()

 * @return array|false an array containing the matched files/directories, an empty array